openpose/setup_local.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
did you end up using this or just use docker?

Reply:
Removed this. I was using docker eventually.